.github/workflows/reusable-labelsync.yml

```diff
@@ -0,0 +1,30 @@
+name: label-sync
+
+on:
+  workflow_call:
+    inputs:
+      operating-system:
+        description: The operating system to use (default `ubuntu-latest`)
+        required: false
+        default: ubuntu-latest
+        type: string
+
+jobs:
+  label-sync:
+    name: Label sync
+    runs-on: ${{ inputs.operating-system }}
+    permissions:
+      contents: read
+      issues: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          sparse-checkout: .github/labels.yml
+          sparse-checkout-cone-mode: false
+          repository: Compairifuel/.github
+      - name: Run Label Sync
+        uses: crazy-max/ghaction-github-labeler@de749cf181958193cb7debf1a9c5bb28922f3e1b # v5
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does this work with the secrets of another repository that uses this workflow?

Reply:
Secrets can be defined per repo, company or environment. So if it uses the repo does this still work?

Reply:
> Secrets can be defined per repo, company or environment. So if it uses the repo does this still work?

GITHUB_TOKEN is a secret that is automatically added to every repository and doesn't need to be defined in the settings, so this should work in every repository